kura/org.eclipse.kura.asset.provider/src/main/java/org/eclipse/kura/asset/provider/AssetConstants.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2017 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ *******************************************************************************/
+package org.eclipse.kura.asset.provider;
+
+import org.eclipse.kura.type.DataType;
+
+/**
```

Comment:
You are right and I agreed with you. `AssetConstants` are now very specific to the default `AssetConfiguration`. The reason the duplication existed because GWT Client Parser sometimes failed in parsing Enum and it happened to me sometimes and that's why a bit of duplication existed in client classes. But you can try to import `AssetConstants` from `org.eclipse.kura.asset.provider`. If it works, it would be single point of change as well and I believe this will be a better alternative.